draft-ietf-capport-architecture.xml

```diff
@@ -379,6 +379,8 @@
         indicates that the User Equipment might need to contact the API Server to receive
         updated information.  For instance, this signal might be generated when the end of a
         session is imminent, or when network access was denied.
+        For simplicity, and to reduce the attack surface, all signals SHOULD be considered
```

Comment:
I think this could be contentious, because it does limit the number of use-cases of the signal. In the future, we might decide that the improved UX/functionality of a richer signal will outweigh the risks introduced by the complexity. Maybe we just need to mention that adding more signals would increase the difficulty of responding to attacks without information loss so that people consider it when designing their system.

Reply:
See what you think of my update.

Reply:
I think it's fine -- leaves the door open.